torchvision/prototype/datasets/__init__.py

```diff
@@ -1,11 +1,10 @@
 try:
     import torchdata
-except (ModuleNotFoundError, TypeError) as error:
+except ModuleNotFoundError:
     raise ModuleNotFoundError(
         "`torchvision.prototype.datasets` depends on PyTorch's `torchdata` (https://github.com/pytorch/data). "
-        "You can install it with `pip install git+https://github.com/pytorch/data.git`. "
-        "Note that you cannot install it with `pip install torchdata`, since this is another package."
-    ) from error
+        "You can install it with `pip install --pre torchdata --extra-index-url https://download.pytorch.org/whl/nightly/cpu"
+    ) from None
```

Comment:
What's the reason not to `raise from error` like before?

Reply:
After the removal of the `TypeError`, showing the excepted error adds no information:

```py
try:
    import foo
except ImportError as error:
    raise ModuleNotFoundError("foo is not available") from error
```

```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    import foo
ModuleNotFoundError: No module named 'foo'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 4, in <module>
    raise ModuleNotFoundError("foo is not available") from error
ModuleNotFoundError: foo is not available
```

vs.

```py
try:
    import foo
except ImportError as error:
    raise ModuleNotFoundError("foo is not available")
```

```
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    import foo
ModuleNotFoundError: No module named 'foo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 4, in <module>
    raise ModuleNotFoundError("foo is not available")
ModuleNotFoundError: foo is not available
```

vs.

```py
try:
    import foo
except ImportError as error:
    raise ModuleNotFoundError("foo is not available") from None
```

```
Traceback (most recent call last):
  File "main.py", line 4, in <module>
    raise ModuleNotFoundError("foo is not available") from None
ModuleNotFoundError: foo is not available
```